virttest/qemu_devices/qcontainer.py

```diff
@@ -1384,12 +1384,13 @@ def define_hbas(qtype, atype, bus, unit, port, qbus, pci_bus,
             addr_spec = None
             if scsi_hba == 'lsi53c895a':
                 addr_spec = [8, 16384]
-            elif scsi_hba == 'virtio-scsi-pci':
+            elif scsi_hba.startswith("virtio"):
                 addr_spec = [256, 16384]
-            elif scsi_hba == 'virtio-scsi-device':
-                addr_spec = [256, 16384]
-                pci_bus = {'type': 'virtio-bus'}
-            elif scsi_hba in ('spapr-vscsi', "virtio-scsi-ccw"):
+                if scsi_hba == 'virtio-scsi-device':
+                    pci_bus = {'type': 'virtio-bus'}
+                elif scsi_hba == 'virtio-scsi-ccw':
+                    pci_bus = None
```

Comment:
OK, the code matches the commit message.  I think I know the answer, but let me ask: why does `virtio-scsi-device` has a `pci_bus` while `virtio-scsi-ccw` does not?